src/main/java/org/generationcp/middleware/service/api/user/UserDto.java

```diff
@@ -31,6 +31,8 @@
 	private String password;
 	private Set<CropDto> crops;
 	private Set<String> authorities;
+	private String selectedCropName;
```

Comment:
@lik-83 why is this required? is it really an attribute of UserDto?

Reply:
These attributes are used to show information about the last opened project. Maybe, we can think to create an API to update and get that info.

Reply:
created: https://ibplatform.atlassian.net/browse/IBP-4397